packages/host/app/services/card-service.ts

```diff
@@ -414,6 +417,34 @@ export default class CardService extends Service {
     }
   }
 
+  async searchPrerendered(
```

Comment:
I don't think we should have a public method here. The only legitimate caller is the PrerenderCardSearch component, and it can just carry the implementation inside itself.